### DISCUSSION TITLE
Why i use jinja template to call {{ds}} not return date

### DISCUSSION THREAD

**Discussion by MarsSu0618** [apache/airflow#13008] [Q&A] [answered]
## Problem
i use `{{ ds }}` in my airflow script , is should be return the execution date as YYYY-MM-DD. But it return string.

```
print('{{ ds }}')
```
response
```
{{ ds }} ## it should be YYYY-MM-DD
```

**Comment by potiuk** [comment, marked as answer]
It should be used in context where the string is marked as "template" string. It doesn't work everwhere. You are using it as "print" statement but the templates only work in parameters of the operators that are added to the list of templated fields. 

See https://airflow.apache.org/docs/apache-airflow/stable/concepts.html#jinja-templating

**Reply by jbmeerkat** [reply]
Current correct link is https://airflow.apache.org/docs/apache-airflow/stable/concepts/operators.html#jinja-templating
